components/RouteGuard.tsx

```diff
@@ -26,6 +28,14 @@ export default function RouteGuard({ children }: { children: ReactNode }) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  async function initialAuthCheck() {
+    const isServerAuth = await checkServerAuth();
+    if (!isServerAuth) {
+      authService.logout(apiBase);
```

Comment:
since checkServerAuth is using `fetcher` now it should automatically logout on 401 response. so this `if` is probably unnecessary (maybe test just to be sure)

Reply:
You're right. If statement is removed and pushed!

(explanation) I'm looking at fetcher and it logs out user for 403 and not 401 (but I since it's server side it shouldn't be returning a 401). If it's a 401 it'll throw an error https://github.com/shardeum/validator-gui/blob/687f97e03c9dbaa16711074259ad50af68f8fd5f/hooks/fetcher.ts#L44.